artwork.md

```diff
@@ -0,0 +1,48 @@
+---
+sidebar: false
+layout: AboutPage
+title: openHAB Artwork
+meta:
+  - name: og:title
+    content: openHAB Artwork
+  - name: og:description
+    content: a vendor and technology agnostic open source automation software for your home
+---
+
+<style>
+.big-title {
+  font-family: 'Open Sans', sans-serif;
+  font-size: 2rem;
+  font-weight: 400;
+  text-align: center;
+}
+img.illustration {
+  width: 30%;
+  align: center;
+}
+@media (max-width: 719px) {
+  img.illustration {
+    width: 100%;
+    align: center;
+  }
+}
+</style>
+
+# openHAB Artwork
+
+## Our Logo
+
+Our logo is our most recognizable asset. We therefore ask you to respect our [trademark policy](/about/trademark) when you reuse it - please read through it in detail and do not hesitate to [reach out to us](/about/trademark.html#contact) in case of any doubts.
+
+We provide different versions of our logo, e.g. with or without the "openHAB" text, a vertical and a horizontal version as well as versions for light and dark backgrounds.
+
+The logos are provided in different formats like svg, eps, ai and png. Whereever possible, please prefer the vector formats over png to avoid pixelation.
```

Comment:
```suggestion
The logos are provided in different formats like svg, eps, ai and png. Wherever possible, please prefer the vector formats over png to avoid pixelation.
```

Thought it was a typo, but https://en.wiktionary.org/wiki/whereever did indeed exist! According to this, it is an obsolete spelling though.